python/dgl/heterograph.py

```diff
@@ -3754,14 +3755,87 @@ def out_degrees(self, u=ALL, etype=None):
         else:
             return deg
 
-    def adjacency_matrix(
-        self, transpose=False, ctx=F.cpu(), scipy_fmt=None, etype=None
-    ):
+    def adjacency_matrix(self, etype=None):
```

Comment:
Since there is an API change, so we need to make sure we highlight this in the release note.

Reply:
OK.